core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -161,6 +167,69 @@ angular.module('oppia').directive('adminMiscTab', [
             });
         };
 
+        var convertLatexToSvgFile = function(latexValue) {
+          // @ts-ignore
```

Comment:
Do not do this. /cc @nishantwrp 

Also @nishantwrp -- what can we do to avoid random ts-ignores with no explanation like this? As you see ... they will happen if automation is not put in to guard against them. /cc @vojtechjelinek

Reply:
Hmm, I'm not sure maybe we can add a check to check if there are comments explaining the `@ts-ignore` above it.